server/vulnerabilities/nvd/indexed_cpe_item.go

```diff
@@ -33,3 +42,30 @@ func (i *IndexedCPEItem) FmtStr(s *fleet.Software) string {
 
 	return cpe.BindToFmtString()
 }
+
+var cpeUpdate = regexp.MustCompile(`(\d+\.\d+\.\d+)((?:a|b|rc)\d+)$`)
```

Comment:
We'll want to clean this up for alpha/beta/rc as well but that doesn't affect Python so we can leave this as-is for this scope.